perf/bloom_filter/launch.lua.in

```diff
@@ -0,0 +1,105 @@
+#! @TARANTOOL_BINARY_DIR@/src/tarantool
+
+-- Benchmark is split into two sources: Lua (this one) and C. The reason is
+-- following: on the one hand C API does not provide convenient interface to
+-- create spaces and define its indexes and format. On the other hand Lua may
+-- spoil benchmark results e.g. due to the GC.
+
+local function greetings()
+    io.write("\n*****************************************************************\n")
+    io.write("\n*********          Bloom filter Perf Test            *******\n")
+    io.write("\n*****************************************************************\n")
+end
+
+
+local bench_run_func = "bench.run"
+local bench_init_func = "bench.init"
+local bench_stop_func = "bench.stop"
+local bench_funcs = { bench_run_func, bench_init_func, bench_stop_func }
+local primary_index_name = "pk"
+-- naming is followed in order to Lua pairs() call iterate through them correct
+local spaces_lookup_cost_coeffs_pairs = { read1VeryFast = 0.05, read2Fast = 0.1, read3Middle = 0.5, read4Slow = 0.9, read5VerySlow = 1 }
+
+local function bench_module_load()
+    for _, func_name in pairs(bench_funcs) do
+        box.schema.func.create(func_name, {language = "C"})
+        box.schema.user.grant('guest', 'execute', 'function', func_name)
+    end
+end
+
+local function bench_module_unload()
+    for _, func_name in pairs(bench_funcs) do
+        box.schema.user.revoke('guest', 'execute', 'function', func_name)
+        box.schema.func.drop(func_name)
+    end
+end
+
+local function schema_create()
+    -- made range big enough so we can benefit from working on different levels
+    local range_size_value = 134217728 * 24
```

Comment:
What is the first magic number?

Reply:
I don't remember why I have chosen it.
I wanted to use default value for `vinyl_range_size` that is 1073741824 for Tarantool < 1.10.2. Changed it.